src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  max-width: $calendar-width;
+  gap: $gap-width;
+  padding: 40px $calendar-side-paddings 0;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    @include square($day-size);
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: ease 0.5s;
```

Comment:
`ease` is a default value for `transition-timing-function`, so you can avoid it, but you should specify the transition properties. In this case that would be `background-color` and `transform`